docs/formation/bootstrap.md

```diff
@@ -0,0 +1,42 @@
+# Governance Bootstrap
+
+In March 2024, the CCC community identified a need for improved organizational structure to better enable contributions to the standard.
+
+This improvement work is being done in two phases, detailed below.
+
+Because this work includes the creation of strict charters and limitations on future change, it will only happen this way once. All subsequent changes must follow steps outlined in the respective group charters.
+
+# Phase 1 — Governance Bootstrapping
+
+The first, and most important step, is to create a charter and assign members to the top level organizational unit: the Steering Committee. This must be done through a consensus of the most active CCC participants from financial service institutions, with an opportunity for input from the entire community.
+
+Process:
+
+1. A proposed charter was be presented to an off-calendar call
```

Comment:
"was be"? Should this say "A proposed charter was presented on an off-calendar call"